class ActionError(Exception):
    pass


class SituationError(Exception):
    pass
